project.yml

```diff
@@ -46,17 +46,33 @@ targets:
           - armv7
         UISupportedInterfaceOrientations:
           - UIInterfaceOrientationPortrait
-          - UIInterfaceOrientationPortraitUpsideDown
-          - UIInterfaceOrientationLandscapeLeft
-          - UIInterfaceOrientationLandscapeRight
+        UIAppFonts:
+          - Inter-Black.ttf
+          - Inter-BlackItalic.ttf
+          - Inter-Bold.ttf
+          - Inter-BoldItalic.ttf
+          - Inter-ExtraBold.ttf
+          - Inter-ExtraBoldItalic.ttf
+          - Inter-ExtraLight.ttf
+          - Inter-ExtraLightItalic.ttf
+          - Inter-Italic.ttf
+          - Inter-Light.ttf
+          - Inter-LightItalic.ttf
+          - Inter-Medium.ttf
+          - Inter-MediumItalic.ttf
+          - Inter-Regular.ttf
+          - Inter-Semibold.ttf
+          - Inter-SemiboldItalic.ttf
+          - Inter-Thin.ttf
+          - Inter-ThinItalic.ttf
```

Comment:
I would suggest adding `UIUserInterfaceStyle: "Light"` to force light theme because it doesn't look great with dark mode

Reply:
good catch!